streaming/java/streaming-api/src/main/java/org/ray/streaming/api/stream/DataStream.java

```diff
@@ -36,16 +36,16 @@ public DataStream(DataStream input, StreamOperator streamOperator) {
   /**
    * Apply a map function to this stream.
    *
-   * @param mapFunction The map function.
-   * @param <R> Type of data returned by the map function.
+   * @param mapFunction The MAP function.
```

Comment:
Unexpected change.

If you cannot do a revert commit I'd suggest you redo the commit so that you don't need to "refactor" state value classes.

Reply:
Yes, I have pushed a brand-new branch instead of reverting the old one.